app/src/main/java/org/systers/mentorship/viewmodels/TaskViewModel.kt

```diff
@@ -0,0 +1,45 @@
+package org.systers.mentorship.viewmodels
+
+import android.arch.lifecycle.ViewModel
+import org.systers.mentorship.MentorshipApplication
+import org.systers.mentorship.R
+
+/**
+ * This class represents the [ViewModel] used for Tasks Screen
+ */
+class TasksViewModel: ViewModel() {
+    var taskList = mutableListOf<String>()
+    var completedTaskList = mutableListOf<String>()
+
+    /**
+     * The function helps in fetching all the tasks from backend
+     */
+    fun getTasks() {
+        taskList.add(MentorshipApplication.getContext().getString(R.string.model_task_1))
+        taskList.add(MentorshipApplication.getContext().getString(R.string.model_task_2))
+        taskList.add(MentorshipApplication.getContext().getString(R.string.model_task_3))
+        //TODO: Fetch the list from the backend
+    }
+
+    /**
+     * The function helps in adding new tasks to the backend
+     */
+    fun addTask(taskName: String) {
+        taskList.add(taskName)
+        //TODO: Update the backend
+    }
+
+    /**
+     * The function helps in updating completed tasks in the backend
```

Comment:
- you're missing 2 `@param` attributes here in this kotlin docs
- rewrite the description to avoid mentioning the backend